src/transport.cpp

```diff
@@ -369,12 +363,37 @@ namespace quicr {
         SendCtrlMsg(conn_ctx, buffer);
     }
 
+    void Transport::SendFetch(ConnectionContext& conn_ctx,
+                              uint64_t subscribe_id,
+                              const FullTrackName& tfn,
+                              messages::ObjectPriority priority,
+                              messages::GroupOrder group_order,
+                              messages::GroupId start_group,
+                              messages::GroupId end_group)
+    {
+        MoqFetch fetch;
+        fetch.subscribe_id = subscribe_id;
+        fetch.track_namespace = tfn.name_space;
+        fetch.track_name.assign(tfn.name.begin(), tfn.name.end());
+        fetch.priority = priority;
+        fetch.group_order = group_order;
+        fetch.start_group = start_group;
+        fetch.start_object = 0; // Request the start of the group
```

Comment:
This works for our media use-case where media is kind of worthless without the start of group and the remaining objects in the group. In the case of other apps, they may want to fetch specific ranges of objects.